cn0534/README.md

```diff
@@ -5,14 +5,19 @@ The main script is based on the `power_check.c` script found on the [plutosdr_sc
 
 Running first the build script is necessary. Prior to running, set the build environment first in the `./build_arm.sh`, change the CC_DIR to the path of the bin folder of the compiler.
 
-For Linux, with the cross compiler set on the host pc and linked with the sysroot used by the pluto, run `./build_arm.sh`. With this build, the binary file can now be stored in a flash drive together with the `runme0.sh` and `parameters.txt`. And upon pluto power up, the binary test file automatically starts and the test is now running.
```

Comment:
this commit can be squashed with the previous since it is added in this PR alone.

Reply:
Squashed this commit to the previous.